src/Common/Core/Impl/Imaging/IImageService.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core.Imaging {
+    /// <summary>
+    /// Provides access to images and icons
+    /// </summary>
+    public interface IImageService {
+        /// <summary>
+        /// Provides image based on one of the predefined types
+        /// </summary>
+        /// <returns>Platform-specific image source</returns>
+        object GetImage(ImageType imageType);
+
+        /// <summary>
+        /// Returns image source given name of the image moniker
+        /// such as name from http://glyphlist.azurewebsites.net/knownmonikers
+        /// when running in Visual Studio.
+        /// </summary>
+        /// <returns>Platform-specific image source</returns>
+        object GetImage(string name);
```

Comment:
These comments have been left in the previous PR, but I still don't understand why we need to differentiate `object GetImage(string name)` and `object GetImage(ImageType imageType)`. Even in `ImageService` implementation, there is this part:
```cs
                case ImageType.File:
                case ImageType.Document:
                    return GetImage("Document");
```

To me it looks like some VS-specific differentiation that matters only to the implementation, but not to the contract. If there is a collision of names, we can always resolve this explicitly, but I don't think there should be two different results for `GetImage(ImageType.Keyword)` and `GetImage("Keyword")` - this is quite confusing